### ISSUE TITLE
How two Unet with two input image

### ISSUE THREAD

**Issue by nopaixx** [keras-team/keras#11060]
Hello

I have two inputs image same size, both image produce same output mask, 
#SIMPLE UNET
```
def get_unet():
    inputs = Input((img_rows, img_cols, 1))
    conv1 = Conv2D(32, (3, 3), activation='relu', padding='same')(inputs)
    conv1 = Conv2D(32, (3, 3), activation='relu', padding='same')(conv1)
    pool1 = MaxPooling2D(pool_size=(2, 2))(conv1)
    drop1 = Dropout(0.2)(pool1)

    conv2 = Conv2D(64, (3, 3), activation='relu', padding='same')(drop1)
    conv2 = Conv2D(64, (3, 3), activation='relu', padding='same')(conv2)
    pool2 = MaxPooling2D(pool_size=(2, 2))(conv2)
    drop2 = Dropout(0.2)(pool2)
    '''''''
    return model

model.fit(imgs_train, imgs_mask_train, batch_size=32, epochs=100, verbose=2, shuffle=True, validation_split=0.1, callbacks=[model_checkpoint])
```
#END SIMPLE MODE WORKING FINE

Now im tring to run with two input image
`  model.fit([imgs_train, imgs_other_train], imgs_mask_train, batch_size=32, epochs=100, verbose=2, shuffle=True, validation_split=0.1, callbacks=[model_checkpoint])`

Im really new to this... im no idea how i can solve, any one have some idea who i need to focuse to solve?

**Comment by mountainML** [comment]
@nopaixx I'm interested in your issue; I have it too.
Have you solved this problem?

**Comment by MohammedAdelFahmi** [comment]
You can add the two images as channels for your input say you have two images of size(265,256,1), you can add them together to make an imagr of (256,256,2) and then you can use unet directly
